Part_One/PartOne_WeatherData.py

```diff
@@ -0,0 +1,71 @@
+
+import json
+
+WEATHER_FILE_NAME = 'weather.json'
+NOT_VAlID_NAME_MSG = 'Not valid name or integers for'
+DAY_OUTPUT_MSG = 'The day number with the smallest temperature spread is'
+NOT_VALID_ROW_MSG = 'There is not a valid row in the file'
+EMPTY_STRING = ''

```

Comment:
This is a bit redundant to use a constant for the empty string literal.
The use of constant is for long literals like the messaged above, or for often typed literals that are easy to typo, or if the literal might change and I want to change it in one place instead of changing it all over the code.